lib/system/dollars.nim

```diff
@@ -7,15 +7,8 @@ proc `$`*(x: int): string {.magic: "IntToStr", noSideEffect.}
   ## spelling `toString`:idx:.
 
 template dollarImpl(x: uint | uint64, result: var string) =
-  type destTyp = typeof(x)
-  if x == 0:
```

Comment:
Well maybe it was a micro optimization？

Reply:
it doens't make much sense now that we avoid the `newString(60)` ; this just adds 2 branches in the general case for no good reason imo